### PR TITLE
Add tax rates to checkout prices calculation

### PR DESCRIPTION
**Pull request by mateuszgrzyb**
I want to merge this change because I want to integrate shipping tax rate and line tax rate into new implementation of checkout prices calculation.

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by mateuszgrzyb** [comment]
Depends on #8103

**Review by IKarbowiak — APPROVED** [review]

File: saleor/checkout/calculations.py [hunk 1]
```diff
@@ -110,24 +146,22 @@ def checkout_line_total(
     checkout_info: "CheckoutInfo",
     lines: Iterable["CheckoutLineInfo"],
     checkout_line_info: "CheckoutLineInfo",
+    address: Optional["Address"] = None,
```
Comment (korycins) [inline saleor/checkout/calculations.py:149]: Do we need it? Previously we were taking an address from checkout_info, which looked like an ok solution.

File: saleor/checkout/calculations.py [hunk 2]
```diff
@@ -179,7 +235,7 @@ def fetch_checkout_prices_if_expired(
     address: Optional["Address"] = None,
     discounts: Optional[Iterable["DiscountInfo"]] = None,
     force_update: bool = False,
-) -> "Checkout":
+) -> Tuple["CheckoutInfo", Iterable["CheckoutLineInfo"]]:
     """Fetch checkout prices with taxes.
 
     Apply checkout prices with taxes from plugins and
```
Comment (korycins) [inline saleor/checkout/calculations.py:241]: I wouldn't say explicitly from webhooks. First we try the "old" approach, where any tax lines are calculated separatly, then we call a "new" approach, where we call single method which will return all prices fully calculated.

**Review by korycins — COMMENTED** [review]
Missing docstring in some new functions.

**Review by fowczarek — APPROVED** [review]

**Pull request merged**